### PR TITLE
Fe 1348 update

### PR DESCRIPTION
**Pull request by chrisamti**

### CODE REVIEW

File: docker/alpine/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN /bin/sh -c 'make test'
 RUN /bin/sh -c 'make build'
 
 # build image, based on alpine
-FROM alpine:3.17
+FROM alpine:3.18
```
Comment (ericfunktxgroup) [inline docker/alpine/Dockerfile]: alpine 3.19?

File: docker/alpine/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # compile
-FROM golang:1.20.5-alpine3.18 as builder
+FROM golang:1.21.8-alpine3.18 as builder
```
Comment (ericfunktxgroup) [inline docker/alpine/Dockerfile]: is there an alpine 3.19 image? - i can see that 3.19 offers a slightly newer version 
https://pkgs.alpinelinux.org/packages?name=*libimagequant*&branch=v3.19&repo=&arch=&maintainer=
4.2.2-r0 

compared to 
https://pkgs.alpinelinux.org/packages?name=*libimagequant*&branch=v3.18&repo=&arch=&maintainer=
4.2.0-r0

File: docker/amazonlinux/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM amazonlinux:2 as rust
+FROM amazonlinux:2023 as rust
```
Comment (ericfunktxgroup) [inline docker/amazonlinux/Dockerfile]: are you sure ? - 2023 is fedora - aka dnf package manager - vs :2 which is still RHEL 7 based

File: docker/alpine/Dockerfile (hunk 1 above)
Comment (chrisamti) [reply docker/alpine/Dockerfile]: Sure...

File: docker/alpine/Dockerfile (hunk 2 above)
Comment (chrisamti) [reply docker/alpine/Dockerfile]: Lets check it out...

File: docker/amazonlinux/Dockerfile (hunk 3 above)
Comment (chrisamti) [reply docker/amazonlinux/Dockerfile]: rolling back

**Review by ericfunktxgroup — APPROVED** [review]

**Pull request merged**